interface-definitions/firewall.xml.in

```diff
@@ -383,6 +383,26 @@
                 </children>
               </node>
               #include <include/firewall/common-rule.xml.i>
+              <leafNode name="ip-length">
+                <properties>
+                  <help>Payload size in bytes, including any extension header</help>
+                  <valueHelp>
+                    <format>u32:1-65535</format>
+                    <description>Numbered packet length</description>
+                  </valueHelp>
+                  <valueHelp>
+                    <format>&lt;start-end&gt;</format>
+                    <description>Packet length range (e.g. 1001-1005)</description>
+                  </valueHelp>
+                  <valueHelp>
+                    <format> </format>
+                    <description>\n\n  Multiple values can be specified as a comma-separated list.\n  For example: '64, 512,1001-1005'</description>
```

Comment:
Please do NOT use empty `<format>` blocks to add additional text. If multiple values are supported, please use `<multi/>`